cupy/cuda/memory.pyx

```diff
@@ -166,10 +167,11 @@ cdef class _Chunk:
 
     def __init__(self, *args):
         # For debug
-        self._init(*args)
+        mem, offset, size, stream_ptr = args
```

Comment:
Why don't you directly write `__init__(self, mem, offset, size, stream_ptr)`?

Reply:
BTW, it looks this method is only used for debugging. Is it related to the performance improvement?

Reply:
This is to avoid compile error because `_init` is a `cdef` method.